contributing/core/vscode-debugger.md

```diff
@@ -6,22 +6,22 @@ The Next.js monorepo provides configurations in the [`.vscode/launch.json`](../.
 
 The common configurations are:
 
-- Launch app development: Run `next dev` with an attached debugger
-- Launch app build: Run `next build` with an attached debugger
-- Launch app production: Run `next start` with an attached debugger
+- **Launch app**: Run `next dev`, `next build`, or `next start` in a directory of your choice, with an attached debugger.
+- **Launch current directory**: Run `next dev`, `next build`, or `next start` in the directory of the currently active file, with an attached debugger.
+- **Run e2e test**: Run an e2e test using the currently active file, with an attached debugger.
 
 ### Run a specific app
 
-Any Next.js app inside the monorepo can be debugged with these configurations. For example to run "Launch app development" against `examples/hello-world`:
+Any Next.js app inside the monorepo can be debugged with these configurations.
 
-1. Open the [`.vscode/launch.json`](../../.vscode/launch.json) file.
-2. Find the configuration "Launch app development".
-3. Edit the `runtimeArgs` array's last item to be `"examples/hello-world"`.
-4. Save the file.
-5. Now you can start the debugger and it will run against the `examples/hello-world` app!
+1. Use the status bar, or the "Run and Debug" item in the Activity Bar, to select the "Launch app" launch configuration.
+2. Enter the app dirname, e.g. `examples/hello-world` or `test/e2e/app-dir/app`.
+3. Select the `next` command from the presented options (`dev`, `build`, or `start`).
 
-To see the changes you make to the Next.js codebase during development, you can run `pnpm dev` in the root directory, which will watch for file changes in `packages/next` and recompile the Next.js source code on any file saves.
+To see the changes you make to the Next.js codebase during development, you can run `NEXT_SERVER_EVAL_SOURCE_MAPS=1 pnpm dev` in the root directory, which will watch for file changes in `packages/next` and recompile the Next.js source code on any file saves.
```

Comment:
Should we set `NEXT_SERVER_EVAL_SOURCE_MAPS` by default in `pnpm dev`? Seems like a better default for dev since it's equally slow for build and faster for rebuild: https://webpack.js.org/configuration/devtool/

Reply:
Yeah, that's a good idea.